legal-api/report-templates/template-parts/notice-of-articles/headerDetails.html

```diff
@@ -0,0 +1,38 @@
+ <div class="business-details-container">
+      <div class="business-details">
+         <table class="business-details-table" role="presentation">
+            <tr>
+               <td>
+                  <table class="business-info" role="presentation">
+                     <tr>
+                        <td class="section-sub-title">Incorporation Number:</td>
+                        <td>{{business.identifier}}</td>
+                     </tr>
+                     <tr>
+                        <td class="section-sub-title">Issued Date and Time:</td>
+                        <td>{{effective_date_time}}</td>
+                     </tr>
+                     <tr>
+                        <td class="section-sub-title">Recognition Date and Time:</td>
```

Comment:
Do you know if Recognition Date and Time is only for IA/NOA, or also for COA/COD?

Reply:
Recognition date is only for IA and NOA